Fabric.Identity.API/scripts/Alter-Config.ps1

```diff
@@ -0,0 +1,47 @@
+# This script will be moved into the install-identity script
```

Comment:
Will need a way to specify which tenant should also go into the Identity web.config